### ISSUE TITLE
Sending large chunk of code may be interrupted in error

### ISSUE THREAD

**Issue by renkun-ken** [randy3k/radian#112]
For example, sending the following code as a whole in vscode to radian will result in error:

```r
tests <- list(
  test1 = c("a", "b", "c"),
  test2 = c("x", "y", "z"),
  test3 = c("b", "c", "d"),
  test4 = c("z", "y", "x"),
  test5 = c("1", "2", "3"),
  test6 = c("p", "q", "r"),
  test7 = c("m", "n", "o"))

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")
```

The output is as follows:

```
R version 3.6.1 (2019-07-05) -- "Action of the Toes"
Platform: x86_64-apple-darwin15.6.0 (64-bit)

> tests <- list( 
    test1 = c("a", "b", "c"), 
    test2 = c("x", "y", "z"), 
    test3 = c("b", "c", "d"), 
    test4 = c("z", "y", "x"), 
    test5 = c("1", "2", "3"), 
    test6 = c("p", "q", "r"), 
    test7 = c("m", "n", "o"))                                                                                                         

>                                                                                                                                     

> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            

>                                                                                                                                     

> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            

>                                                                                                                                     

> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            

>                                                                                                                                     

> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            
paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,"/* test2 
>                                                                                                                                     
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf*/", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 
> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 
>                                                                                                                                     
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/**/", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 
> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/*/", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 
>                                                                                                                                     
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$
*/", sprintf("`%s` double NULL,", tests$test6),
> cat(  "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            
*/", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
Error: unexpected ',' in:"`%s` double NULL,", tests$test3),
"  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
),"

>                                                                                                                                     
), collapse = "\n"), file = "tests.txt")

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")
^[[18;1R^[[18;13R
> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/* test2 */", sprintf("`%s` double NULL,", tests$test2), 
    "/* test3 */", sprintf("`%s` double NULL,", tests$test3), 
    "/* test4 */", sprintf("`%s` double NULL,", tests$test4), 
    "/* test5 */", sprintf("`%s` double NULL,", tests$test5), 
    "/* test6 */", sprintf("`%s` double NULL,", tests$test6), 
    "/* test7 */", sprintf("`%s` double NULL,", tests$test7) 
  ), collapse = "\n"), file = "tests.txt")                                                                                            

cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")
^[[18;1R^[[18;13R
>                                                                                                                                     
cat(paste0(c(
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")
^[[18;1R^[[18;13R
> cat(paste0(c( 
    "/* test1 */", sprintf("`%s` double NULL,", tests$test1), 
    "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  
  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  "/  "/  "/*/  */", sprintf
  ("`%s` double NULL,", tests$test6),                                                                                                 
  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/* test2 */", sprintf("`%s` double NULL,", tests$test2),
  "/* test3 */", sprintf("`%s` double NULL,", tests$test3),
  "/* test4 */", sprintf("`%s` double NULL,", tests$test4),
  "/* test5 */", sprintf("`%s` double NULL,", tests$test5),
  "/* test6 */", sprintf("`%s` double NULL,", tests$test6),
  "/* test7 */", sprintf("`%s` double NULL,", tests$test7)
), collapse = "\n"), file = "tests.txt")
^[[18;1R^[[18;13RError: unexpected '*' in:
"  "/* test1 */", sprintf("`%s` double NULL,", tests$test1),
  "/  "/  "/*/  "/  "/  "/*"

>   "/* test7   "/* test2 */", sprintf("`%s` double NULL,", tests$test2),                                                             
Error: unexpected '*' in "  "/* test7   "/*"

>   "/* test3 */", sprintf("`%s` double NULL,", tests$test3),                                                                         
Error: unexpected ',' in "  "/* test3 */","

>   "/* test4 */", sprintf("`%s` double NULL,", tests$test4),                                                                         
Error: unexpected ',' in "  "/* test4 */","

>   "/* test5 */", sprintf("`%s` double NULL,", tests$test5),                                                                         
Error: unexpected ',' in "  "/* test5 */","

>   "/* test6 */", sprintf("`%s` double NULL,", tests$test6),                                                                         
Error: unexpected ',' in "  "/* test6 */","

>   "/* test7 */", sprintf("`%s` double NULL,", tests$test7)                                                                          
Error: unexpected ',' in "  "/* test7 */","

> ), collapse = "\n"), file = "tests.txt")                                                                                            
Error: unexpected ')' in ")"

>  
```

**Comment by renkun-ken** [comment]
When I'm using iTerm2 under macOS, I copy paste the code to `R` and it does not work but `radian` works.

And in vscode, both not working.

**Comment by renkun-ken** [comment]
It is caused by the same problem as I replied at https://github.com/randy3k/radian/issues/111#issuecomment-531151369. Now it works.

The only problem here is that the major part of the code sent to radian disappears but executed anyway.

<img width="509" alt="image" src="https://user-images.githubusercontent.com/4662568/64849049-226a7380-d645-11e9-9b71-65cb98a83f69.png">

I added `cat(n, "\n")` before each chunk to ensure all are executed.
